test/e2e/kafka/00-assert.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     size:
       plan: startup-2
     zone: ch-dk-2
-    version: '3.2'
+    version: '3.5'
```

Comment:
is it anyhow related?

Reply:
Somewhat, the e2e-test doesn't work anymore with 3.2.